### DISCUSSION TITLE
2023-07-10

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#15] [daily_meeting]
### 데일리 미팅
마코, 노아, 룩소 - 3시 근로
공통 - 1시 30분 수업

### 오늘의 할 일

- 공통

  - MVP 줄이기

- 백엔드

  - CI/CD
 
  - 기능 명세

  - API 명세

- 프론트엔드

  - CI/CD(엽토)

  - 디자인(룩소, 노아)

### 내일 할 일

- 준과의 면담

- 기획 확정 후에 와이어 프레임 스케치

**Comment by nlom0218** [comment]
### 체크인

10점 - 주말에 푹 쉬니까 에너지가 충전된 기분이네요:)

### 체크아웃

10점 - 하루스터디의 스터디 너무 재밌어요 👍

### 하루스터디의 스터디를 하며 느낀점

1. 빠른 피드백으로 다음 사이클 때 다시 한 번 목표를 재조정하여 목표를 도달하기 위해 노력한다.
2. 정해진 짧은 시간에 목표를 도달하기 위해 집중한다.
3. 목표에 도달하지 못했을 때의 아쉬움이 다음 번에 잘해야겠다는 것으로 이어진다.
4. 짧은 목표를 정하는 것이 생각보다 어렵다. 이를 잘 설정할 수 있도록 유도해야겠다.
5. 20분이라는 짧은 시간도 좋지만 시간을 스터디장이 자유롭게 커스튬하면 좋을 것 같다. 
6. 쉬는시간도 함께
7. 간단한 규칙도 함께 제사히면 좋을 것 같다.

**Comment by yeopto** [comment]
### 체크인

9점 - 컨디션 좋은데 너무 덥다.

### 체크아웃

10점 - 생각보다 기획 변경이 스무스하게 진행되어서 +1

### 우리가 직접 짧은 단위로 스터디 해본 소감

확실히 짧은 시간 단위로 목표 설정하고 학습하니 집중도 잘 되고, 내가 설정한 목표를 달려가기 위해 더 노력하는 것 같아서 효율적이었다고 생각한다.

**Comment by woo-jk** [comment]
### 체크인
8점 - 주말동안 블로그 쓰려고 했는데 걍 놀아버렸다

### 체크아웃
10점 - 스터디 방식 좋은것 같다고 느낌!

### 스터디에 대한 느낀점
사이클이 기존 방식보다 짧으니 집중이 잘됐다. 또한 사이클 전에 추상적인 목표가 아닌, 구체적인 목표를 정함으로써 스터디 중간에 방향성을 잃지 않았다. 스터디에 대한 피드백이 빨라서 좋다.

**Comment by jaehee329** [comment]
### 체크인
10점 - 비 오고 나니 기온은 좀 떨어져서 좋네요~

### 체크아웃
9점 - 벌써 조금 피곤해졌네요

### 스터디 하며 느낀 점
 예상처럼 컴팩트 하고 딱 좋은 것 같아요.
이번에는 다른 주제로 진행했지만 같은 주제로 진행하더라도 괜찮을 것 같은 느낌?
오늘 진행자 테오의 역할이 우리가 개발하는 툴의 기능이 되면 딱일 것 같아요

**Comment by woosung1223** [comment]
### 체크인
8점 - 사람이 많아 라떼를 못사서 -2점

### 체크아웃
9점 - 오늘 스터디를 빡시게 해서 기분이 좋습니다

### 하루스터디 직접 경험해보고 느낀점
생각보다 더 효율적이라는 생각이 들었습니다. 짧은 시간에 짧은 주제를 빠르게 학습할 수 있었고, 빠른 피드백을 얻을 수 있었습니다.


### KPT
K - 디스커션을 통해 토론하는게 정말 생각 정리에 좋았습니다
P - 없을 무
T - 앞으로 논쟁거리가 생기면 디스커션을 바로 열어보자!

**Comment by aak2075** [comment]
### 체크인

10점 - 기분 좋게 하루를 시작 해봐요

### 체크아웃

10점 - 스터디하니까 집중 잘되네요

### 스터디를 하면서 느낀 점

시간이 짧으니까 딴짓하기 어렵고 집중이 잘 된다.
잠깐씩 집중해서 스터디 하기는 좋지만 목표 설정 등 생각하는 시간도 있어서 이걸로 계속 하기는 어려울 수 있겠다?
